microhapdb/tests/test_cli.py

```diff
@@ -361,6 +363,7 @@ def test_hg37(capsys):
 mh18PK-87558 MHDBM-1e5374f1    GRCh37 chr18 1960543,1960558,1960562,1960567,1960583,1960589 2.3659 0.1325 0.0494 10.1016/j.fsigen.2018.05.008
 mh18USC-18pA MHDBM-56dfa93b    GRCh37 chr18         5280016,5280019,5280069,5280070,5280092 3.4330 0.2130 0.1818 10.1016/j.fsigen.2019.102213
   mh18CP-005 MHDBM-a85754d3    GRCh37 chr18                 8892862,8892891,8892894,8892905 3.6722 0.0904 0.0059                       ALFRED
+ mh18ZHA-004 MHDBM-db649dba    GRCh37 chr18             14315930,14315951,14315957,14316045 2.9579 0.1949 0.0118 10.1016/j.fsigen.2020.102255
```

Comment:
`mh18ZHA-004` is a new marker that has been added that is contained within the query of this test. It has been updated to include the new marker.